great_expectations/init_notebooks/create_expectations.ipynb

```diff
@@ -151,7 +190,7 @@
    "outputs": [],
    "source": [
     "# this is how you can see which data batch was loaded\n",
```

Comment:
# Reviewing batch_kwargs shows what arguments were used by the datasource to load data